cclib/parser/orcaparser.py

```diff
@@ -1384,6 +1388,149 @@ def energy_intensity(line):
+                # STEOM-CCSD RESULTS
+                # ------------------
+                #
+                # IROOT=  1:  0.120159 au     3.270 eV   26371.8 cm**-1
+                #   Amplitude    Excitation
+                #    0.104201    61 ->  76
+                #    ...
+                #   -0.362075    69 ->  73
+                #   Ground state amplitude:  0.000000
+                # 
+                # Percentage Active Character     99.28
+                # 
+                #   Amplitude    Excitation in Canonical Basis
+                #   -0.158598    64 ->  72
+                #   ...
+                #   -0.111587    69 ->  75
+                # 
+                # IROOT=  2:  0.123788 au     3.368 eV   27168.4 cm**-1
+                #
+                # or:
+                #
+                #  ----------------------
+                #  EOM-CCSD RESULTS (RHS)
+                #  ----------------------
+                #
+                #   IROOT=  1: -0.001688 au    -0.046 eV    -370.5 cm**-1
+                #     Amplitude    Excitation
+                #     -0.693399     x ->  70
+                #   Percentage singles character=     93.46
+                #       
+                #   IROOT=  2:  0.061276 au     1.667 eV   13448.5 cm**-1
+                etenergies.append(float(line.split()[6]))
+                if self.mdci_et_mult is not None:
+                    etsyms.append(self.mdci_et_mult)
+                sec = []
+                # Header line.
+                # There is no header for CIS.
+                if not cis:
+                    line = next(inputfile)
+                # First orbital contribution line.
+                line = next(inputfile)
+                while "->" in line:
+                    coeff_split = line.split()
+                    # CIS prints coefficients after the orbitals, other modules are reversed.
+                    if not cis:
+                        contrib = coeff_split[0]
+                        start = coeff_split[1]
+                        end = coeff_split[3]
+                    else:
+                        # CIS looks like this:
+                        # 32 ->  37    0.037502 (-0.193653)
+                        # 34 ->  35    0.935796 ( 0.967365)
+                        # 34 ->  36    0.014167 (-0.119024)
+                        # Note that sometimes the coefficient has whitespace between the brackets, sometimes not.
+                        start = coeff_split[0]
+                        end = coeff_split[2]
+                        contrib = "".join(coeff_split[4:])[1:-1]
+                    
+                    try:
+                        # TODO: Unrestricted?
+                        sec.append([(int(start), 0), (int(end), 0), float(contrib)])
+                        
+                    except ValueError:
+                        # Sometimes we come across lines like:
+                        # 0.690372    69 -> x
```

Comment:
I'm really not sure. This notation is shown in two places in the manual (one for high level excited states like we're seeing here and the second for ionisation). I suspect this notation was originally for ionisation-like processes and has since been reused in a different context.

These results are the pre-calculations before the final excited states are calculated (presumably at some lower-level of theory). My best guess is that the types of excited states being calculated at this stage either don't have the same concept of starting and final orbital transitions, and/or that the orbitals being transitioned to or from do not map well to the ground state orbitals.